app/models/miq_request.rb

```diff
@@ -420,11 +420,27 @@ def task_check_on_execute
   def execute
     task_check_on_execute
 
-    deliver_on = nil
     if get_option(:schedule_type) == "schedule"
-      deliver_on = get_option(:schedule_time).utc rescue nil
+      start_time = get_option(:schedule_time).utc rescue nil
```

Comment:
Note to self what if start_time is in the past (aka scheduled in the future but approval took longer)

Reply:
I would recommend that we refactor this line.  The classic UI sets `get_option(:schedule_time)` to a `ActiveSupport::TimeWithZone` object which supports the `utc` call.

I suspect that if this is set from the API or some other means it would set the time as a string in which case the code will silently error and use `nil`.  It is also a good opportunity to get rid of the inline rescue as well.

Reply:
https://github.com/ManageIQ/manageiq/pull/17547/commits/bb299814bd825c6fea725426b4e412df983ccf15